src/jmh/java/io/deephaven/csv/benchmark/datetimecol/DateTimeColumnBenchmark.java

```diff
@@ -58,29 +58,31 @@ public InputProvider() {
 
     BenchmarkResult<long[]> result;
 
-    @TearDown(Level.Invocation)
     public void check(final InputProvider input) {
         input.tableMaker.check(result.columns());
     }
 
     @Benchmark
     @OperationsPerInvocation(OPERATIONS)
-    public void deephaven(final InputProvider input, final ReusableStorage storage) throws Exception {
+    public Object deephaven(final InputProvider input, final ReusableStorage storage) throws Exception {
```

Comment:
Was there a reason why this is Object and not `BenchmarkResult<long[]>` ?

Reply:
Didn't mean to change this file.